apps/emqx_auth/test/emqx_authz/emqx_authz_api_sources_SUITE.erl

```diff
@@ -582,6 +582,121 @@ t_source_move(_) ->
 
     ok.
 
+t_sources_reorder(_) ->
```

Comment:
nit: for regression purpose, maybe add a disabled source (the enable/disable state should not affect ordering).

Reply:
Done for both authz and authn.